helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/ClusterContext.java

```diff
@@ -35,14 +35,14 @@
  * This class tracks the rebalance-related global cluster status.
  */
 public class ClusterContext {
-  private final static float ERROR_MARGIN_FOR_ESTIMATED_MAX_COUNT = 1.1f;
-
   // This estimation helps to ensure global partition count evenness
   private final int _estimatedMaxPartitionCount;
   // This estimation helps to ensure global top state replica count evenness
   private final int _estimatedMaxTopStateCount;
   // This estimation helps to ensure per-resource partition count evenness
   private final Map<String, Integer> _estimatedMaxPartitionByResource = new HashMap<>();
+  // This estmations helps to ensure global resource usage evenness.
+  private final float _estimatedMaxUtilization;
```

Comment:
projected might be a better word.

Reply:
Same as what I replied previously, I think "estimate" is better.